s3_management/manage.py

```diff
@@ -1,20 +1,21 @@
 #!/usr/bin/env python
 
 import argparse
+import base64
+import dataclasses
```

Comment:
This is a followup to the other comment.
```suggestion
import dataclasses
import functools
```